src/Synchronizer.php

```diff
@@ -126,6 +126,8 @@ private function getHostsLines($container)
         // Networks
         if (isset($container['NetworkSettings']['Networks']) && is_array($container['NetworkSettings']['Networks'])) {
             foreach ($container['NetworkSettings']['Networks'] as $networkName => $conf) {
+                preg_match(sprintf('/^%s_(.+)$/', $this->getProjectName($container)), $networkName, $matches);
+                $networkShortName = isset($matches[1]) ? $matches[1] : null;
```

Comment:
```
                $networkShortName = isset($matches[1]) ? $matches[1] : 'default';
```

should allow to not require setting the name as `DOMAIN_NAME=default:foo.bar` if the default network is used.